@here/harp-vectortile-datasource/lib/VectorTileDataEmitter.ts

```diff
@@ -242,14 +253,20 @@ export class VectorTileDataEmitter {
     private m_maxGeometryHeight: number = 0;
     private m_minGeometryHeight: number = 0;
 
+    private readonly m_gatherFeatureAttributes: boolean;
```

Comment:
Why don't you just store the options object here instead of repeating the instance methods? At least that is something that we seem to do regularly in other classes.

Reply:
makes sense. Wouldn't you rather do this in a separate PR?